src/website/src/pages/ec/components/navigation/inpage-navigation/docs/code.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Showcase
+order: 2
+---
+
+import { Playground, Html } from '@ecl/website-components';
+import inpageNavigationExample from '../demo';
+
```

Comment:
The demo here is not 1:1 to current ECL website. It's acceptable and even preferable if you ask me.
However, it might worth adding a paragraph here to give a hint to the user to the playground link to see a working example in addition to source code.

Reply:
As discussed, I added custom markup example 👍